fixtures/hasura/common/metadata/scalar-types/Int.hml

```diff
@@ -60,3 +60,40 @@ definition:
     enable: true
   graphql:
     typeName: IntComparisonExp
+
+---
+kind: AggregateExpression
+version: v1
+definition:
+  name: IntAggregateExp
+  operand:
+    scalar:
+      aggregatedType: Int
+      aggregationFunctions:
+        - name: _avg
+          returnType: Int
```

Comment:
The return type for averages over `Int` should be `Float`. We need to fix that in the connector capabilities. I filed an issue to follow up on this, https://linear.app/hasura/issue/NDC-385/return-type-for-average-of-ints-should-be-float-not-int